AUTHORS

```diff
@@ -142,4 +142,6 @@ Dmitry Kropachev <dmitry.kropachev@gmail.com>
 Oliver Boyle <pleasedontspamme4321+gocql@gmail.com>
 Jackson Fleming <jackson.fleming@instaclustr.com>
 Sylwia Szunejko <sylwia.szunejko@scylladb.com>
-Karol Baryła <karol.baryla@scylladb.com>
\ No newline at end of file
+Karol Baryła <karol.baryla@scylladb.com>
+Marcin Mazurek <marcinek.mazurek@gmail.com>
+Moguchev Leonid Alekseevich <lmoguchev@ozon.ru>
```

Comment:
Last name mistakenly comes first here; IMO it should be:
`Leonid Alekseevich Moguchev`
Correct?

Reply:
@moguchev  gave me a thumbs up in the PR description, so I thought it was ok. This order was in the author of the commit.

Reply:
Now I am not sure

Reply:
I guess the order of names does not really make any difference here.